libraries/AP_Frsky_Telem/AP_Frsky_Telem.cpp

```diff
@@ -28,18 +28,30 @@
 #include <AP_BattMonitor/AP_BattMonitor.h>
 #include <AP_RangeFinder/AP_RangeFinder.h>
 #include <AP_Common/AP_FWVersion.h>
-#include <GCS_MAVLink/GCS.h>
 #include <AP_Common/Location.h>
 #include <AP_GPS/AP_GPS.h>
 #include <AP_Baro/AP_Baro.h>
+#include <AP_Param/AP_Param.h>
+#include <AP_Logger/AP_Logger.h>
+#include <AP_Vehicle/AP_Vehicle.h>
+#include <AC_Fence/AC_Fence.h>
+
+#include <stdio.h>
 #include <math.h>
 
+#include "mavlite.h"
+
+#define SPORT_TX_PACKET_DUPLICATES 1
+#define SPORT_RX_PACKET_DISCARD_DUPLICATE
+
 extern const AP_HAL::HAL& hal;
+using namespace MAVLITE;
 
 AP_Frsky_Telem *AP_Frsky_Telem::singleton;
 
 AP_Frsky_Telem::AP_Frsky_Telem(bool _external_data) : AP_RCTelemetry(TIME_SLOT_MAX),
-    use_external_data(_external_data)
+    use_external_data(_external_data),
+    _frsky_parameters(&AP::vehicle()->frsky_parameters)
```

Comment:
should use a parameter pointer

Reply:
for this one I need help, can you point me at an example somewhere?